src/typeMapper.js

```
@@ -1,12 +1,22 @@
 import {
-  GraphQLInt,
-  GraphQLString,
+   GraphQLInt,
+   GraphQLString,
    GraphQLBoolean,
    GraphQLFloat,
    GraphQLEnumType,
    GraphQLList
  } from 'graphql';
 
+let _userMappingFunc;
+/**
+ * A function to set a custom mapping of types
+ * @param mapFunc
+ */
+export const mapType = (mapFunc = () => false) => {
+    _userMappingFunc = mapFunc;
+};
```

No reason to silently fail if the user calls it with something like `mapType(Object)` or `mapType(undefined)`